okta/plugin.spec.yaml

```diff
@@ -605,3 +695,37 @@ actions:
         description: The user ID of the Okta user
         type: string
         required: false
+  update_blacklist_zones:
+    title: Update Blacklist Zones
+    description: Block or unblock address or network by adding or removing from a blacklist network zone
+    input:
+      name:
+        title: Name
+        description: Name of blacklist zone
+        type: string
+        required: true
+        example: Insight Connect Blacklist Zone
+      address:
+        title: Address
+        description: IP address, Network range, or CIDR to block or unblock
+        type: string
+        required: true
+        example: 198.51.100.1
+      blacklist_state:
+        title: Blacklist State
+        description: True to block, false to unblock
+        type: boolean
+        required: false
+        default: True
+        example: True
+    output:
+      zone_list:
+        title: Zone List
+        description: Updated zone list
+        type: zone_list
+        required: true
+      success:
```

Comment:
I think we can remove this variable since there's a built-in `$success` - $success is True when no exception has been raised.